modules/common/src/main/base/LilaTypes.scala

```diff
@@ -26,6 +26,11 @@ trait LilaTypes extends ValidTypes {
   def fufail[X](s: String): Fu[X] = fufail(LilaException(s))
   def fufail[X](f: Failures): Fu[X] = fufail(LilaException(f))
   val funit = fuccess(())
+  val fuTrue = fuccess(true)
+  val fuFalse = fuccess(false)
+
+  implicit val fUnitZero: Zero[Fu[Unit]] = Zero.instance(funit)
+  implicit val fBooleanZero: Zero[Fu[Boolean]] = Zero.instance(fuFalse)
```

Comment:
Is this maybe covered by `Fu[Zero]` (with `Zero[Boolean]`)?

Reply:
yeah it is covered by requires a new object for each zero

Reply:
👌

Reply:
and I saw some code in futures that checks for object equality so using the exact same future when possibly may allow scala to collapse some chains